custom_components/tuya_local/devices/nac_ap470_purifier.yaml

```diff
@@ -0,0 +1,146 @@
+name: Air purifier
+products:
+  - id: y7iiubdc5sufjvo3
+    name: "NAC AP470 Air Purifier"
+primary_entity:
+  entity: fan
+  translation_key: fan_with_presets
```

Comment:
When using this key, your preset_mode values should be mapped to keys from the translation file https://github.com/make-all/tuya-local/blob/main/custom_components/tuya_local/translations/en.json#L170

The values currently in there were taken from presets that used to be listed in HA source code, but are no longer there, and I guess in future will become default translations for fan when they get around to translating the fan platform fully. "sleep" is the only direct mapping, but "smart" should be a good substitute for Auto, and "baby", "normal", "fresh", "strong"  could be a progression for the other levels.

Reply:
I've removed translation_key: fan_with_presets, and will leave names as it was. They are the same as in tuya app.